custom-plugin-rules/markdown-validator/rule-validate-markdown.js

```diff
@@ -0,0 +1,77 @@
+module.exports = ValidateMarkdown
+const markdownlint = require("markdownlint");
+const config = {
+  // the list is here https://github.com/DavidAnson/markdownlint#rules--aliases
+  MD013: {line_length: 120},
+  MD041: false, // first line should be h1
+  MD047: false, // should end with newline
+}
+
+function checkString(description, ctx) {
+  let options = {
+    "strings": {
+      "desc": description
+    },
+    "config": config
+  };
+  markdownlint(options, function callback(err, result) {
+    if (!err) {
+      // if there's no problem do nothing
+      if (result.desc.length) { // desc is the key in the options.strings object
+        let lines = description.split("\n");
+
+        result.desc.forEach((desc) => {
+          message = desc.ruleDescription;
+          // add line number context for longer entries
+          if (desc.lineNumber > 1 ) {
+            message = message + " (near: " + lines[desc.lineNumber].substring(0,20) + "... )";
+          }
+
+          ctx.report({
+            message: message,
+            location: ctx.location.child('description'),
+          });
+        });
+
+      }
+    }
+  });
+}
+
+function ValidateMarkdown() {
+  console.log("OpenAPI Markdown: validate");
+  return {
+   Info: {
+      enter(target, ctx) {
+        if(target["description"]) {
+          return checkString(target["description"], ctx);
```

Comment:
I'd rewrite here and below as the following:

```suggestion
      enter({ description }, ctx) {
        if(description) {
          return checkString(description, ctx);
```

Reply:
I applied this, but it trips up where no description is present. The linter tells me "description is not defined", so I think I probably need to keep the old syntax so I can check if the field is there.

Reply:
Never mind, I took it out and put it back and it seems fine now ...